deepcell_toolbox/metrics.py

```diff
@@ -182,11 +182,17 @@ def __init__(self,
                                  y_pred.shape, y_true.shape))
 
         # Relabel y_true and y_pred so the labels are consecutive
-        y_true, _, _ = relabel_sequential(y_true)
-        y_pred, _, _ = relabel_sequential(y_pred)
+        y_true_relabeled, _, _ = relabel_sequential(y_true)
+        y_pred_relabeled, _, _ = relabel_sequential(y_pred)
 
-        self.y_true = y_true
-        self.y_pred = y_pred
+        if not (np.array_equal(y_true, y_true_relabeled) and
+                np.array_equal(y_pred, y_pred_relabeled)):
```

Comment:
Is it possible that one is equal and the other is not?

Reply:
Yeah, for example if the true_labels were sequentially relabeled during preprocessing, but the predicted labels were not relabeled

Reply:
Would we want to warn in that case?

Reply:
Yes, anytime any of the images have to be relabeled, it results in a mismatch between the label map before and the label map after. This means that you can't use the metrics.predictions, which tells you which cells were misclassified, since the cell ids will be different.